include/xeus-cpp/xeus_cpp_config.hpp.in

```diff
@@ -10,9 +10,12 @@
 #define XEUS_CPP_CONFIG_HPP
 
 // Project version
-#define XEUS_CPP_VERSION_MAJOR 0
-#define XEUS_CPP_VERSION_MINOR 4
-#define XEUS_CPP_VERSION_PATCH 0
+#define XEUS_CPP_VERSION_MAJOR @XEUS_CPP_VERSION_MAJOR@
+#define XEUS_CPP_VERSION_MINOR @XEUS_CPP_VERSION_MINOR@
+#define XEUS_CPP_VERSION_PATCH @XEUS_CPP_VERSION_PATCH@
```

Comment:
This should not be configured by an external tool. This makes the cpp code depend on the building tools. This should be reverted to the previous behavior which was the correct one.